src/components/collect-button.sass

```diff
@@ -64,15 +78,26 @@
 
       &.heterozygote
         border-radius: $collect-button-inner-border-radius
-        border: $collect-button-border-width dashed $color-key-blue
+        border: $collect-button-border-width dotted $color-key-blue
 
     .icon
       height: $collect-button-icon-height
 
     .label
-      font-size: $font-size-label1
-      color: $color-nav-cinder-dark2
+      font-size: $font-size-label2
+      color: $color-nav-cerulean-dark2
+
+.collect-button-holder:hover
+  cursor: pointer
+
+.collect-button-holder:hover .collect-button-outline
+  background-color: $color-nav-cerulean-dark1
+
+.collect-button-holder:hover .collect-button-outline.uncollected
+  background-color: $color-nav-cinder-light3
+
+.collect-button-holder:hover .x-close
+  background-color: $color-nav-cerulean-dark1
 
-  .collect-button:hover
-    border-color: $color-nav-cerulean-dark2
-    cursor: pointer
\ No newline at end of file
+.collect-button-holder:hover .x-close.uncollected
+  background-color: transparent
```

Comment:
@mlewandowski Random question (trying to merge in Eli's branch):
is this
```
.collect-button-holder:hover .x-close.uncollected
  background-color: transparent
```
just to clear the prior effects of highlighting `.collect-button-holder:hover .x-close` when it's unselected? Or is it actually important that it is transparent?

If it's the former, could we also just change lines 99-100 to be
```
.collect-button-holder:hover .x-close:not(.uncollected)
  background-color: $color-nav-cerulean-dark1
```

Or, alternatively, could we just not render the `.x-close` in the empty button at all? Is there any reason for it?

Reply:
I know you already got to this, but, yes, everything you said is true.  Not rendering the x-close is best.